### ISSUE TITLE
Weirdness in 1.0.0 release?

### ISSUE THREAD

**Issue by rafeememon** [Wikiki/bulma-timeline#14]
I bumped the version of `bulma-timeline` in my project to `1.0.0`, but something seems a little off.  For instance, I don't see this change in the release: https://github.com/Wikiki/bulma-timeline/pull/13  In `package.json`, the version is indeed `1.0.0` but `main` points at `index.js`:

```
  ...

  "main": "index.js",
  "name": "bulma-timeline",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/Wikiki/bulma-timeline.git"
  },
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "version": "1.0.0"
}
```

I may have done something weird but I wanted to flag it just in case.

**Comment by rafeememon** [comment]
If I had to take a guess, since [this commit](https://github.com/Wikiki/bulma-timeline/commit/f87be1fb15a9e8109d2569ceb89e11cb3ef9bb6f) is in the release, I think that feature branch may have been used for the release instead of `master`.  But that's just a complete shot in the dark =)

**Comment by Wikiki** [comment]
Hi @rafeememon ,

I've published a v1.0.1 and check both timeline.sass and packahge.json are good.

Let me know if you still having troubles.

Regards
